assets/scripts/test_sdk.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# This script follows steps written in README.md
+# Using the rust file specified as the unique argument.
+# Call at the top directory of the nexus-zkvm project
+# For example:
+# ./assets/scripts/test_sdk.sh examples/src/bin/fib3.rs
+# Every command needs to succeed
+set -e
+
+ORIGINAL_DIR=$(pwd)
+PROJECT_NAME="/tmp/nexus-host-ci"
+
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 <file.rs>"
+    exit 1
+fi
+
+if [ ! -f "$1" ]; then
+    echo "Usage: $0 <file.rs>"
+    echo "where <file.rs> is a path to a file."
+    exit 1
+fi
+
+if [ -e "$PROJECT_NAME" ]; then
+    echo "Error: Directory '$PROJECT_NAME' already exists."
+    exit 1
+fi
+
+set -x
+
+cleanup() {
+    cd "$ORIGINAL_DIR"
+    rm -rf "$PROJECT_NAME"
+}
+
+trap cleanup SIGINT
+
+error_handler() {
+    echo "Error occurred in smoke.sh on line ${1}. Exiting."
```

Comment:
Here's a leftover file name `smoke.sh` to replace.

Reply:
thank you. Fixed in a8c718f